README.md

```diff
@@ -149,7 +174,29 @@ The ancestors have provided an example of one route; you do the other six!
 GET '/warriors/:id'
   * This is the show route, which finds a warrior by ID, and displays information about that warrior.
 
-Replace this with your answer
+
+  1. To display list of soliders:
+  GET-/soldiers-soldiers#index
+
+  2. to return an HTML form to create a new soldier:
+  GET-/soldiers/new-soldiers#new
+
+  3. To create a new soldier:
+  POST-/soldiers-soldiers#create
+
+  4. To display a specific soldier:
+  GET-/soldiers/:id-soldiers#show
+
+  5. To return an HTML form for editing a soldier:
+  GET-/soldiers/:id/edit-soldiers#editing
+
+  6. To return an HTML form for editting a soldier:
+  PATCH-/soldiers/:id-soldiers#update
+
+  7. To delete a specific soldier:
+  DELETE-/soldiers/:id-soldiers#destroy
```

Comment:
We generally want to express routes VERB "/path/goes/here" we then associate that route with an action written controller#action. You've got all the info right, the formatting just isn't super clear